receiver/windowsperfcountersreceiver/windowsperfcounters_scraper.go

```diff
@@ -64,15 +66,20 @@ func (s *scraper) initialize(ctx context.Context) error {
 
 				c, err := pdh.NewPerfCounter(counterPath, true)
 				if err != nil {
-					errors = append(errors, fmt.Errorf("error initializing counter %v: %w", counterPath, err))
+					errors = append(errors, fmt.Errorf("counter %v: %w", counterPath, err))
 				} else {
 					s.counters = append(s.counters, c)
 				}
 			}
 		}
 	}
 
-	return componenterror.CombineErrors(errors)
+	// log a warning if some counters cannot be loaded, but do not crash the app
+	if len(errors) > 0 {
+		s.logger.Warn("error initializing counters", zap.Error(componenterror.CombineErrors(errors)))
```

Comment:
Technically, this is not an error, so would something like "some counters could not be initialized" be a better message?